reactioncommerce/reaction
# [Marketplace] Owner permission should be transferable #2188

Resolves #2188 (transfer of ownership of a Shop or Marketplace to another administrator)

**How to test:**
* Transfering Marketplace Owner
_ In the default (primary) shop, log in as default admin and invite a user as shop admin (e.g Shop Manager).
_ On the newly invited user, click the group dropdown, and select "Owner"
_ Confirm that there's a pop up warning about the intended change 
_ On proceeding, the current owner should be replaced with the new owner. 
_ NB: The old owner (who is currently logged in) losses the right to change the owner back (since she has transferred ownership)
_ Log in as the new user that the ownership was transferred to (in new browser) 
_ Go to the admin panel for that user, and confirm that you can transfer the ownership back

* Test for another shop
_ Create a new shop. (e.g using the "Invite Owner Form" from the Marketplace tab in Shop settings)
_ Log in as the owner of the shop, and run through a similar process like above
